references/indexes.md

```diff
@@ -44,146 +51,107 @@ Get the index relative information.
 
 ```bash
 $ curl \
-  -X GET 'http://localhost:7700/indexes/12345678'
+  -X GET 'http://localhost:7700/indexes/movie'
 ```
 
 #### Response: `200 Ok`
 
 ```json
 {
-  "name": "Movie",
-  "uid": "12345678",
+  "name": "movie",
+  "identifier": "movie_id",
   "createdAt": "2019-11-20T09:40:33.711324Z",
   "updatedAt": "2019-11-20T10:16:42.761858Z"
 }
 ```
 
-
-
 ## Create an index
 
 <RouteHighlighter method="POST" route="/indexes"/>
 
 Create an index.
 
-The [schema](/guides/main_concepts/indexes.md) definition is optionally send through the body.
-If no [schema](/guides/main_concepts/indexes.md) has been defined when the first document is sent it will be [infered based on that document](/guides/main_concepts/indexes.md#schema-definition).
-
-A randomly generated UID will be returned. It's associated to the new index. This UID will be essential to make all request over the created index.
-
+Takes as parameter a unique `uid` and **optionnaly** the name of the <glossary word="document identifier"/>: `identifier`.
 
 #### Body
 
-The body take at least an index name.
 
-The body accepts an optional [schema](/guides/main_concepts/indexes.md) definition of your documents.
+| Variable  | Description           |
+|-----------|-----------------------|
+| **uid** | The index unique identifier |
+| **identifier** | the document <glossary word="attribute" /> of his unique identifier |
```

Comment:
```suggestion
| **identifier** | The document <glossary word="attribute" /> of its unique identifier |
```

Reply:
And if you chose to use the glossary, instead of the link, you might change it everywhere.